recipes/configs/qwen2/evaluation.yaml

```diff
@@ -0,0 +1,43 @@
+# Config for EleutherEvalRecipe in eleuther_eval.py
+#
+# To launch, run the following command:
+#    tune run eleuther_eval --config qwen2/evaluation
+
+# Model Arguments
+model:
+  _component_: torchtune.models.qwen2.qwen2_7b
+
+# Checkpointer
+checkpointer:
+  _component_: torchtune.training.FullModelHFCheckpointer
+  checkpoint_dir: /tmp/Qwen2-7B-Instruct
+  checkpoint_files: [
+    model-00001-of-00004.safetensors,
+    model-00002-of-00004.safetensors,
+    model-00003-of-00004.safetensors,
+    model-00004-of-00004.safetensors
+  ]
+  output_dir: ./ # Not needed
```

Comment:
👀

Reply:
There's nothing to OUTPUT.

Reply:
Yeah I just mean this is an indication of a gap in the current checkpointer, this should clearly not be a mandatory arg